libnavigation-core/src/androidTest/java/com/mapbox/navigation/core/TripServiceActivityTest.kt

```diff
@@ -0,0 +1,76 @@
+package com.mapbox.navigation.core
+
+import androidx.test.espresso.Espresso.onIdle
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.uiautomator.By
+import androidx.test.uiautomator.Until
+import com.mapbox.navigation.core.test.R
+import com.mapbox.navigation.testing.ui.NotificationTestRule
+import com.schibsted.spain.barista.assertion.BaristaEnabledAssertions.assertEnabled
+import com.schibsted.spain.barista.assertion.BaristaVisibilityAssertions.assertContains
+import com.schibsted.spain.barista.assertion.BaristaVisibilityAssertions.assertDisplayed
+import com.schibsted.spain.barista.interaction.BaristaClickInteractions.clickOn
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+internal class TripServiceActivityTest :
+    NotificationTestRule<TripServiceActivity>(TripServiceActivity::class.java) {
+
+    @Before
+    fun setup() {
+        onIdle()
+    }
+
+    @Test
+    fun checkStartButtonAccessible() {
+        R.id.toggleNotification.let {
+            assertDisplayed(it)
+            assertEnabled(it)
+            assertContains(it, "Start")
+        }
+    }
+
+    @Test
+    fun checkNotificationViewContent() {
+        R.id.notifyTextView.let {
+            assertContains(it, "")
+        }
+
+        R.id.toggleNotification.let {
+            clickOn(it)
+        }
+
+        R.id.notifyTextView.let {
+            assertDisplayed(it)
+            assertContains(it, "Time elapsed: + ")
+        }
+    }
+
+    @Test
+    fun checkNotificationContent() {
+        R.id.toggleNotification.let {
+            clickOn(it)
+        }
+
+        R.id.notifyTextView.let {
+            assertDisplayed(it)
+            assertContains(it, "Time elapsed: + ")
+        }
```

Comment:
Do we need this? See above comment: this belongs to `notifyTextView` test (not the `TripNotification` view content) and actually is already tested in `checkNotificationViewContent` https://github.com/mapbox/mapbox-navigation-android/blob/a0481ac141df032300606bb2575d619eef0ac297/libnavigation-core/src/androidTest/java/com/mapbox/navigation/core/TripServiceActivityTest.kt#L47-L50 I think we can safely remove.

Reply:
removed tests not related to TripService